README.md

```diff
@@ -1 +1,129 @@
 # Universal-Manipulator
+# Abstract
+
+A mechanical arm, programmable with similar functions to a human arm having. The gripper attached can be used to pick objects from one  given coordinates and drop to another given coordinates .
+
+# Team Members
+* Vipin Kumar
```

Comment:
@shubhamsharele write the names in alphabetical order

Reply:
and also add their github lik in it

Reply:
I have made the necessary changes and none of my group members has a GitHub profile